Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'capistrano-s3'
+gem 'dotenv-rails'
```

Comment:
These are great reminders - I still need to flesh out the README more but have been hesitant due to the flux.